types/module/configurator.go

```diff
@@ -38,3 +57,23 @@ func (c configurator) MsgServer() grpc.Server {
 func (c configurator) QueryServer() grpc.Server {
 	return c.queryServer
 }
+
+// RegisterMigration implements the Configurator.RegisterMigration method
+func (c configurator) RegisterMigration(moduleName string, fromVersion uint64, handler func(store sdk.KVStore) error) error {
```

Comment:
Let's update the doc string with more details about the `fromVersion` or maybe we can rename to `forVersion`? Let's make it clear that a migration is valid for version `fromVersion` and it updates to version `fromVersion+1` iiuc.

Reply:
These docs are actually on the public interface method. I renamed to forVersion.